### ISSUE TITLE
Compatibility

### ISSUE THREAD

**Issue by Macfresno** [LordGood/Ponies-Ascendant#2]
Hello,

I've seen this mod in the forum and looks amazyng, but it does not seems to work on Alpha 22 Venustas, it's any plan to work on the compatibility issues with the current version?

Regards.

**Comment by LordGood** [comment]
I'm afraid this repo is truncated to 0 AD A18. The mod council tries to keep this mod up to date in their repository but rarely is it useable. I would recommend the old release for this mod

**Comment by heuzef** [comment]
Is an update planned for futur?

**Comment by StanleySweet** [comment]
Yes, soon ! but it will be here https://github.com/0ADMods/ponies_ascendant
